src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1004,6 +1050,7 @@ export default class GeFormRenderer extends LightningElement{
     reset() {
         this.resetFormState();
         this._openedGiftId = null;
+        this.clearErrors();
```

Comment:
+++ doing a similar thing in #6404

Reply:
@psadie-sfdo @sokosaurus Are we going to have conflicts/overlaps to untangle between this PR and #6404 do you think?

Reply:
I've tried to shuffle some things around to minimize conflicts but I wouldn't be surprised if there were a few remaining things that we'd need to ultimately untangle here.